src/components/Header/mobile-menu/menu-trigger/index.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from "react";
+import HamburgerIcon from "./icons/hamburger-icon.inline.svg";
+import CloseIcon from "./icons/close-icon.inline.svg";
```

Comment:
Are icons from [RescUI](https://rescui.dev/#/Components/Icons) not suitable?

Reply:
They do, fixed